### DISCUSSION TITLE
Morphological Segmentation of words

### DISCUSSION THREAD

**Discussion by MaveriQ** [explosion/spaCy#12378] [Language Support]
Hi there. I am not sure if this is the right category for this question, but since it pertains to language models in general, I thought I would put it here. 

Presently spacy has morphological analysis support for various languages. I was wondering if there is a way to use that for splitting words into their morphological components, so for example "paratrooper" will become "para", "troop", "er".. etc. 

Thank you.

**Comment by thomashacker** [comment]
Hey, thanks for your question.
Currently, the [morphologizer](https://spacy.io/api/morphologizer#_title) does not support any features for splitting tokens based on their morphological components.

**Reply by MaveriQ** [reply]
Thanks
